www/views/faq.html

```diff
@@ -76,7 +76,7 @@ <h4>Can I use it with Angular?</h4>
       </section>
       <section id="author" class="qa">
         <h4>Who is responsible for Cyclops?</h4>
-        <p>Cyclops is written and maintained by the UI/UX Team at <a href="https://www.ctl.io">CenturyLink Cloud</a>. To get in touch, <a href="https://github.com/CenturyLinkCloud/Cyclops/issues">file an issue</a> in the <a href="https://github.com/CenturyLinkCloud/Cyclops">Github repo</a>, or request access to their <a href="https://centurylinkuiux.slack.com/">Slack</a> by emailing cloud-design [a] ctl.io.</p>
+        <p>Cyclops is written and maintained by the UI/UX Team at <a href="https://www.ctl.io">CenturyLink Cloud</a>. To get in touch, <a href="https://github.com/CenturyLinkCloud/Cyclops/issues">file an issue</a> in the <a href="https://github.com/CenturyLinkCloud/Cyclops">Github repo</a>, or request access to their <a href="https://teams.microsoft.com/l/channel/19%3A38e4f3fc16634e11b9e3c732271fa273%40thread.skype/Cyclops?groupId=8631295d-43fc-4e40-93e5-dc8c9ab4d127&tenantId=72b17115-9915-42c0-9f1b-4f98e5a4bcd2">Microsoft Team Cyclops Channel</a> by emailing cloud-design [a] ctl.io.</p>
```

Comment:
No need to request access to Microsoft Teams. The link will prompt for their CTL credentials. Please change to the following:

`<p>Cyclops is written and maintained by the UI/UX Team at <a href="https://www.ctl.io">CenturyLink Cloud</a>. To get in touch, <a href="https://github.com/CenturyLinkCloud/Cyclops/issues">file an issue</a> in the <a href="https://github.com/CenturyLinkCloud/Cyclops">Github repo</a>, or talk Cyclops in our <a href="https://teams.microsoft.com/l/channel/19%3A38e4f3fc16634e11b9e3c732271fa273%40thread.skype/Cyclops?groupId=8631295d-43fc-4e40-93e5-dc8c9ab4d127&tenantId=72b17115-9915-42c0-9f1b-4f98e5a4bcd2">Microsoft Teams Cyclops Channel</a></p>`